deflect/qt/QmlStreamer.cpp

```diff
@@ -66,6 +66,11 @@ QQmlEngine* QmlStreamer::getQmlEngine()
     return _impl->getQmlEngine();
 }
 
+Stream *deflect::qt::QmlStreamer::getStream()
```

Comment:
I would like to see how it's used. From my naive point of view, I would like to send the command directly from QML, hence it should be provided as a function/signal in Qt. Exposing the stream here means wiring the QML action to the sendCommand outside somewhere. But I have not enough knowledge at this point.